source/code/plugins/out_oms_blob.rb

```diff
@@ -115,12 +115,17 @@ def get_blob_uri(container_type, data_type, custom_data_type, suffix)
     #   msgs: string[]. messages
     def append_blob(uri, msgs)
       if msgs.size == 0
-        return true
+        return 0
```

Comment:
the return value was not used before. I now change it to return the size of the data.